vscode/src/extension.ts

```diff
@@ -238,3 +240,17 @@ export class QsTextDocumentContentProvider
     return getLibrarySourceContent(uri.path);
   }
 }
+
+function checkForOldQdk() {
+  const extension = vscode.extensions.getExtension(
+    "quantum.quantum-devkit-vscode"
+  );
+
+  const warningMessage =
+    "Previous QDK Extension `quantum.quantum-devkit-vscode` found. We recommend uninstalling the prior QDK before using this release.";
+
+  if (extension) {
+    log.warn(warningMessage);
+    vscode.window.showErrorMessage(warningMessage);
```

Comment:
`showWarningMessage` feels like a more appropriate level for this one.

Reply:
Since we're not `await`ing here, could you add a clarifying comment saying we don't wait for the dialog to be dismissed before continuing? I think it's not 100% clear this is intentional.